src/App.tsx

```diff
@@ -20,20 +25,132 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    itemWidth: 130,
+    frameSize: 3,
+    step: 3,
+    animationDuration: 1000,
+    infinity: false,
+  };
+
+  handlerEvent = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
  handleEvent = (event: React.ChangeEvent<HTMLInputElement>) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364